app/code/core/Mage/Newsletter/etc/system.xml

```diff
@@ -118,6 +118,28 @@
                         </un_email_template>
                     </fields>
                 </subscription>
+                <security translate="label">
+                    <label>Security</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>1</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <enable_form_key translate="label">
+                            <label>Enable Form Key Validation</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment><![CDATA[<strong style="color:red">Important!</strong> Enabling this option means
+                            that your custom templates used for newsletter subscription contain form_key output.
```

Comment:
I'm not sure how linefeeds are handled in CDATA and translation, can it be multiple lines?

```suggestion
                            that your custom templates for newsletter subscription must contain the <em>form_key</em> post parameter.
```

Reply:
I think it's a bad idea and must be a single line.
Because we will have:
```
"<strong style=""color:red"">Important!</strong> Enabling this option means
                            that your custom templates used for newsletter subscription contain form_key output.
                            Otherwise newletter subscription will not work.","Translation..."
```

Reply:
I saw this comment under `System -> Admin -> Security -> Enable Form Key Validation On Checkout`, and I figured to add it for this flag as well just as a notice.
> ```diff
> - form_key output.
> + form_key</em> post parameter.
> ```
form_key is this case is meant to be the name of the block that outputs the hidden input. So I think we could keep this part unchanged? or do you have another suggestion?

Reply:
If the comment already exist, keep it unchanged to get benefit existing translations.

Reply:
@luigifab the comment isn't translated

Reply:
oh sorry, I misread, I'm going out